### PR TITLE
Change web app url.

### PR DESCRIPTION
**Pull request by AnnStepanova**

### CODE REVIEW

**Review by AleksandrHorev — APPROVED** [review]

**Comment by AleksandrHorev** [comment]
@AnnStepanova please review

File: README-ru.md [hunk 1]
```diff
@@ -30,7 +30,7 @@
 Воспользуйтесь один из 2 способов:
 * Поиском по содержимому строки в файле перевода (удобно когда строка большая или уникальная)
 * **Через веб-приложение TON-Surf**  
-  В [веб-приложении TON-Surf](https://web.ton.surf/) есть помощник локализатора - он показывает полный путь до нужной строчки при наведении на неё.
+  В [веб-приложении TON-Surf](https://beta.ton.surf/accounts) есть помощник локализатора - он показывает полный путь до нужной строчки при наведении на неё.
```
Comment (BigRoma) [inline README-ru.md]: I think you don't need **accounts** at the end.

**Comment by AleksandrHorev** [comment]
@AnnStepanova @AleksandrHorev please rerview

**Review by AleksandrHorev — APPROVED** [review]

**Review by BigRoma — CHANGES_REQUESTED** [review]
Please fix the link and we can merge it.

**Review by BigRoma — APPROVED** [review]

**Pull request merged**